civihr_default_theme/scss/mail.scss

```diff
@@ -55,19 +55,28 @@ table.social {
 
 }
 .social .soc-btn {
```

Comment:
Padding: I have changed it to adjust the height of button in all email client except the outlook.
Display: I have changed it because now we are using table structure to wrap `<a>` 
Min-Width: is used for outlook.
Width: is used to adjust the button width for all eamil client except the outlook.